--- camlimages-2.11/configure.in~	Sat May  3 23:59:08 2003
+++ camlimages-2.11/configure.in	Sun May  4 00:00:23 2003
@@ -296,26 +296,26 @@
 if test "$NOEXTERNAL" = "false" -a "X$SUPPORT_FREETYPE" = "X" ; then
     SUPPORT_FREETYPE="false"
     AC_CHECK_LIB(ttf, TT_Init_FreeType,
-  	    AC_CHECK_HEADER(freetype.h,
+  	    [AC_CHECK_HEADER(freetype.h,
   		    AC_DEFINE(HAVE_FREETYPE) LIBFREETYPE="-lttf"; 
   		    SUPPORTED="$SUPPORTED freetype(1)";
   		    SUPPORT_FREETYPE="true", 
-  	    AC_CHECK_HEADER(/usr/include/freetype/freetype.h,
+  	    [AC_CHECK_HEADER(/usr/include/freetype/freetype.h,
   		    AC_DEFINE(HAVE_FREETYPE) LIBFREETYPE="-lttf"; 
   		    SUPPORTED="$SUPPORTED freetype(1)";
   		    SUPPORT_FREETYPE="true";
   		    INCLUDE_FREETYPE="-I/usr/include/freetype", 
-  	    AC_CHECK_HEADER(/usr/local/include/freetype.h,
+  	    [AC_CHECK_HEADER(/usr/local/include/freetype.h,
   		    AC_DEFINE(HAVE_FREETYPE) LIBFREETYPE="-lttf"; 
   		    SUPPORTED="$SUPPORTED freetype(1)";
   		    SUPPORT_FREETYPE="true";
   		    INCLUDE_FREETYPE="-I/usr/local/include/", 
-  	    AC_CHECK_HEADER(/usr/local/include/freetype/freetype.h,
+  	    [AC_CHECK_HEADER(/usr/local/include/freetype/freetype.h,
   		    AC_DEFINE(HAVE_FREETYPE) LIBFREETYPE="-lttf"; 
   		    SUPPORTED="$SUPPORTED freetype(1)";
   		    SUPPORT_FREETYPE="true";
   		    INCLUDE_FREETYPE="-I/usr/local/include/freetype", 
-  		    SUPPORT_FREETYPE="false")))))
+  		    SUPPORT_FREETYPE="false")])])])])
 fi
 
 if test "X$SUPPORT_FREETYPE" = "X"; then
